app/views/settings/preferences/other/show.html.haml

```diff
@@ -38,6 +38,17 @@
                    required: false,
                    wrapper: :with_label
 
+    .fields-row
+      .fields-group.fields-row__column.fields-row__column-6
+        = ff.input :default_quote_policy,
+                   collection: %w(public followers nobody),
```

Comment:
Not critical for this PR, because I realize many of the other "constant" arrays in `UserSettings` are not like this ... but possible change here and/or future refactor to that class would be to bump more of these collection arrays to constants in appropriate places? (ie, elsewhere in this view template we refer to `Status.selectable_visibilities`, for example)